### ISSUE TITLE
Retiring Project that Provided Similar Functionality

### ISSUE THREAD

**Issue by jovanni-hernandez** [jupyter-server/jupyter-scheduler#348]
## Description

A few years ago I developed a plugin for Jupyter, [JupyterLab Scheduler](https://github.com/tiburon-security/jupyterlab_scheduler/), that provided functionality for scheduling the execution of notebooks. Over the years, I've kept the plugin up to date, with the assistance of a few other open source contributors who also found the plugin useful. While performing some recent updates, one of our users discovered that an official scheduler had been released and brought it my attention, so I'd like to retire my project and start pointing users who stumble on my project to the official repo. @dlqqq indicated that I should open a new issue to formalize this transition.

**Comment by dlqqq** [comment]
@jovanni-hernandez Thank you for opening this issue! I mainly wanted to start a discussion on the next steps that should be taken to gracefully retire your project and direct users here. Let's agree on what needs to be done first. I can get us started here. These steps should be followed in top-down order:

- Push a commit to add a notice that the project has been retired in favor of jupyter-scheduler. This notice should go at the top of the README.md for visibility.
- Perform a minor release on NPM and PyPi to update the README.md on the NPM and PyPi pages. This would require a version bump from `0.1.5` to `0.1.6`.
- Archive the repository on GitHub.
- Deprecate the [NPM package](https://www.npmjs.com/package/@tiburon-security/jupyterlab_scheduler)
- Deprecate the [PyPi package](https://pypi.org/project/jupyterlab-scheduler/)

The steps for deprecating packages on NPM and PyPi should be fairly straightforward and google-able; let me know if you'd like me to help on this.

Next steps for you:

1. Review this list and see if you have any comments or additions.
2. Once you approve the list of retirement steps, you can create a PR that updates the README.md, and cc me on it for a review.
3. Once that PR is merged, you can perform the rest of the steps in order.

Thank you again for taking the initiative on this effort! 🤗

**Comment by rddaz2013** [comment]
**Both are great projects** - thx for the work you all put into this, both with similar functions but with exceptions.
For  jupyter-scheduler I am missing the **'endless' cronjob** method without the job logging thing - the problem is the same #335 /#328 . In my case we are talking about 217000 calls for the job (7 months) for that an auto clean for jobs would be very nice.

René

Sure I can use crontab manually on a python script - but I think per GUI has a nice touch.

**Comment by dlqqq** [comment]
@rddaz2013 Absolutely, no question. I want to emphasize that we are not trying to coerce maintainers to deprecate their packages in favor of ours; the retirement path we are proposing is purely voluntary and can be withdrawn at any time.

Once we hear back from the lead maintainer and get confirmation that we will move forward with retiring jupyterlab-scheduler, we will assign the issues you mentioned higher priority to make the migration as painless as possible for existing users. 🤗
